resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -89,6 +89,9 @@ aws.settings.aws_cli_settings=AWS CLI Settings
 aws.settings.clouddebug.location=cloud-debug CLI executable:
 aws.settings.dynamic_resources_configurable.clear_all=Clear All
 aws.settings.dynamic_resources_configurable.select_all=Select All
+aws.settings.dynamic_resources_configurable.suggest_types.dialog.message=Please suggest additional AWS resource types you would like to see (e.g. AWS::S3::Bucket).\n(max length: 2000 chars)
+aws.settings.dynamic_resources_configurable.suggest_types.dialog.title=Additional Resource Type Suggestions
+aws.settings.dynamic_resources_configurable.suggest_types.prompt=Something missing?
```

Comment:
nit: Probably long but would "Want more resource types?" be clearer?

Reply:
possibly but that messes with the alignment! I'm open to someone more skilled in UI than me to help!

Reply:
I like the current wording. Short and sweet.

Reply:
The wording seems good to me. As far as alignment goes, you'd want to move the link + buttons to a column across all rows instead of their own rows. That's assuming you want the buttons to shift down if the text wraps. Need to make sure the margin between the link and buttons matches the default row margin too. Otherwise the current layout makes sense (and is less work...)